biogeochem/EDCanopyStructureMod.F90

```diff
@@ -278,40 +302,44 @@ subroutine canopy_structure( currentSite , bc_in )
           if(patch_area_counter > max_patch_iterations .and. area_not_balanced) then
              write(fates_log(),*) 'PATCH AREA CHECK NOT CLOSING'
              write(fates_log(),*) 'patch area:',currentpatch%area
-             do i_lyr = 1,z
-                write(fates_log(),*) 'layer: ',i_lyr,' area: ',arealayer(i_lyr)
-                write(fates_log(),*) 'rel error: ',(arealayer(i_lyr)-currentPatch%area)/currentPatch%area
-                write(fates_log(),*) 'abs error: ',arealayer(i_lyr)-currentPatch%area
-             enddo
+             write(fates_log(),*) 'fraction that is imperfect (unclosed):',imperfect_fraction
              write(fates_log(),*) 'lat:',currentSite%lat
              write(fates_log(),*) 'lon:',currentSite%lon
              write(fates_log(),*) 'spread:',currentSite%spread
-             currentCohort => currentPatch%tallest
-             do while (associated(currentCohort))
-                write(fates_log(),*) 'coh ilayer:',currentCohort%canopy_layer
-                write(fates_log(),*) 'coh dbh:',currentCohort%dbh
-                write(fates_log(),*) 'coh pft:',currentCohort%pft
-                write(fates_log(),*) 'coh n:',currentCohort%n
-                write(fates_log(),*) 'coh carea:',currentCohort%c_area
-                ipft=currentCohort%pft
-                write(fates_log(),*) 'maxh:',prt_params%allom_dbh_maxheight(ipft)
-                write(fates_log(),*) 'lmode: ',prt_params%allom_lmode(ipft)
-                write(fates_log(),*) 'd2bl2: ',prt_params%allom_d2bl2(ipft)
-                write(fates_log(),*) 'd2bl_ediff: ',prt_params%allom_blca_expnt_diff(ipft)
-                write(fates_log(),*) 'd2ca_min: ',prt_params%allom_d2ca_coefficient_min(ipft)
-                write(fates_log(),*) 'd2ca_max: ',prt_params%allom_d2ca_coefficient_max(ipft)
-                currentCohort => currentCohort%shorter
+             do i_lyr = 1,z
+                write(fates_log(),*) '-----------------------------------------'
+                write(fates_log(),*) 'layer: ',i_lyr,' area: ',arealayer(i_lyr)
+                write(fates_log(),*) 'abs error (layer-patch): ',(arealayer(i_lyr)- &
+                     (1._r8-imperfect_fraction)*currentPatch%area)
```

Comment:
Did you want to make this the absolute error?
```suggestion
                write(fates_log(),*) 'abs error (layer-patch): ',abs(arealayer(i_lyr)- &
                     (1._r8-imperfect_fraction)*currentPatch%area)
```

Reply:
ahh, no, I should leave it as the bias. will fix